doc/decoding/searchlight.rst

```diff
@@ -4,85 +4,83 @@
 Searchlight : finding voxels containing information
 ===========================================================
 
+This page overviews searchlight analyses and how they are approached
+in nilearn with the `nilearn.decoding.SearchLight` estimator.
+
 .. currentmodule:: nilearn.decoding
 
+.. contents:: **Contents**
+    :local:
+    :depth: 1
+
 Principle of the Searchlight
 ============================
 
-Searchlight was introduced in `Information-based functional brain mapping
-<http://www.pnas.org/content/103/10/3863>`_, Nikolaus Kriegeskorte,
-Rainer Goebel and Peter Bandettini (PNAS 2006) and consists in scanning the
-images volume with a *searchlight*. Briefly, a ball of given radius is
-scanned across the brain volume and the prediction accuracy of a
-classifier trained on the corresponding voxels is measured.
-
-Preparing the data
-====================
-
-Loading
--------
-
-Fetching the data from internet and loading it can be done with the
-provided functions (see :ref:`loading_data`):
+Searchlight analysis was introduced in `Information-based functional brain 
+mapping <http://www.pnas.org/content/103/10/3863>`_ (Kriegeskorte et al., 
+2006), and consists of scanning the brain with a *searchlight*. Briefly, a 
+ball of given radius is scanned across the brain volume and the prediction 
+accuracy of a classifier trained on the corresponding voxels is measured. 
 
-.. literalinclude:: ../../examples/02_decoding/plot_haxby_searchlight.py
-    :start-after: # Load Haxby dataset
-    :end-before: # Restrict to faces and houses
+Searchlights are also not limited to classification; regression (e.g., 
+`Kahnt et al, 2011
+<https://www.sciencedirect.com/science/article/pii/S0896627311002960?via%3Dihub>`_) 
+and representational similarity analysis (e.g., `Clarke and Tyler, 2014 
+<https://www.jneurosci.org/content/34/14/4766.short>`_) are other uses of 
+searchlights. Currently, only classification and regression are supported 
+in nilearn.
 
-Reshaping the data
--------------------
+.. topic:: **Further Reading**
+    
+    For a critical review on searchlights, see `Etzel et al (2013) 
+    <https://www.sciencedirect.com/science/article/pii/S1053811913002917>`_.
 
-For this example we need:
 
-- to put X in the form *n_samples* x *n_features*
-- compute a mean image for visualization background
-- limit our analysis to the `face` and `house` conditions
-  (like in the :ref:`introduction to decoding <decoding_intro>`)
+Preparing the data
+==================
 
-.. literalinclude:: ../../examples/02_decoding/plot_haxby_searchlight.py
-    :start-after: # Restrict to faces and houses
-    :end-before: # Prepare masks
+:class:`SearchLight` requires a series of brain volumes as input, `X`, each with 
```

Comment:
Does this reference properly ? Or do we need to give the full:

`:class:nilearn.decoding.SearchLight` ?

Reply:
We can write it like this: `:class:~nilearn.decoding.SearchLight` .
This will form the proper link and only show the final element (SearchLight) as the text.